proc.go

```diff
@@ -95,6 +95,10 @@ func (p Proc) CmdLine() ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
+	
+	if len(data) < 1 {
+		return nil, fmt.Errorf("could not read cmdline pid:%d", p.PID)
```

Comment:
Okay, I'm building some kind of system event listener including docker and kernel proc events. 
As a part of the listener, I found this repository for proc fs reader. 
And we're planning to use prometheus for our system metric server. :)